docs(windows): update kb0076 to include log capturing on firefox (#2569)
keymanapp/help.keyman.com: knowledge-base/kb0076.md

Reviewer: ```suggestion
4. Exit Keyman for Windows and Keyman Developer, if they are running, as well as any application where you need to demonstrate the problem.
```

Reviewer: ```suggestion
5. Restart Keyman for Windows and the target application.
```

> # HOWTO: Run Debugging in Keyman Desktop and Firefox

Reviewer: I think we should change it to "Keyman for Windows" instead of "Keyman Desktop" (throughout this doc).

```suggestion
# HOWTO: Run Debugging in Keyman for Windows and Firefox
```

> Keyman Support may ask you to run debugging in Keyman Desktop or Keyman Developer. The debug report assists us in resolving complex support issues. Please follow the steps below to generate the debug report.

Reviewer: ```suggestion
Keyman Support may ask you to run debugging in Keyman for Windows or Keyman Developer. The debug report assists us in resolving complex support issues. Please follow the steps below to generate the debug report.
```

10. Send us a copy of your debug file(s). You'll find them, in the folder above, named system##.etl or system##.log (where ## is a number).
11. Please include notes to us about the issue, including what you typed, what output you expected to get (if possible!), and the results you actually received.

## Capture a log in Firefox

> #### Only follow this if the issue is related to Google products or other web applications.

Reviewer: I find this sentence confusing. Why the focus on Google products? Shouldn't I follow these steps if I have problems typing on a normal website? And in general, when should I not follow these steps?

Reviewer 2: Yes I think remove this sentance or it should just say 

"These steps are for issues you have that are reproducilbe using Firefox"

MengHeng the main times I have asked for this debug capture in the forums is when users have had problems using Firefox and Googe Docs/Sheets. However, I also used it for just contentedable fields in Firefox (so general web pages really).

It is most usefull for finding issues with Firefox. However it could be usefull if there is a particullar webpage/application that doesn't work with Keyman and any browser and you could use Firefox just for its abillity to capture a log.

> #### Only follow this if the issue is related to Google products or other web applications.

Author: ```suggestion
#### These steps are for debugging the issues that are reproducible using Firefox.
```

> 1) Open a testcase in a tab, e.g. Google Docs or Sheets

Reviewer 2: ```suggestion
1) Open a testcase in a tab, e.g. Google Docs or Sheets, Website with issue
```

2) Open and write `about:logging` in another tab

3) Choose “Logging to a file” radio button and type a path to save the log into the text field, then, press “Set Log File” button

4) Set “New log modules:” field to KeyboardHandler:4,IMEHandler:4,sync and click “Set Log Module” button.

5) If you see “Start logging” button in top of the page, click it.

> 6) Back to the tab loaded the testcase with mouse

Reviewer 2: ```suggestion
6) Go back to the tab loaded with the testcase using the mouse
```

> 7) Click the editor and do the minimum steps to reproduce the bug

Reviewer 2: ```suggestion
7) Do the minimum steps to reproduce the bug
```

> 8) Back to the tab loaded `about:logging` with mouse

Reviewer 2: ```suggestion
8) Go back to the tab loaded `about:logging` with mouse
```
